Group.sol

```diff
@@ -55,6 +56,7 @@ contract Group {
     Message[] groupChat;
     GroupVote[] groupVotes; //which contains all group votes (might be used even if group is dictatorship)
     GroupMember[] members;
+    BinaryContract[] groupBinaryContracts;
```

Comment:
I would not add that attribute, but rather create one when a GroupContract is expired.
I see that I forgot to add 'uint expirationDate' on the GroupContract, I will add it.